test/localPRG_test.cpp

```diff
@@ -272,6 +275,17 @@ TEST(LocalPRGTest, nodes_along_path) {
     EXPECT_EQ(j, l3.nodes_along_path(p3)[1]->id);
 }
 
+TEST(LocalPRGTest, nodes_along_path_without_memoization) {
+    assert(LocalPRG::path_memoization_in_nodes_along_path_method == false);
```

Comment:
Use the googletest fixture `EXPECT_FALSE` instead of `assert`

Reply:
I guess here we need to disable memoization and test `LocalPRG::nodes_along_path()`.

So I think that this this `assert()` is not correct, `EXPECT_FALSE` won't also forcibly disable memoization. So in the next commit, we just forcibly disable it and test the method.